errors.go

```diff
@@ -88,22 +72,11 @@ func (errs ErrManifestVerification) Error() string {
 	return fmt.Sprintf("errors verifying manifest: %v", strings.Join(parts, ","))
 }
 
-// ErrUnknownLayer returned when layer cannot be found.
-type ErrUnknownLayer struct {
-	FSLayer manifest.FSLayer
-}
-
-func (err ErrUnknownLayer) Error() string {
-	return fmt.Sprintf("unknown layer %v", err.FSLayer.BlobSum)
-}
-
-// ErrLayerInvalidDigest returned when tarsum check fails.
-type ErrLayerInvalidDigest struct {
+// ErrManifestBlobUnknown returned when layer cannot be found.
+type ErrManifestBlobUnknown struct {
```

Comment:
Perhaps, but we may have other fields here in the future. Mostly, we do this so the type is never treated as a string.